pkg/config/config_template.yaml

```diff
@@ -2807,6 +2807,18 @@ api_key:
 ## Network Devices Configuration ##
 ###################################
 
+## @param network_devices - custom object - optional
+## Configuration related to Network Devices Monitoring
+#
+# network_devices:
+
+  ## @param namespace - string - optional - default: default
+  ## Namespace can be used to disambiguate devices with the same IP.
+  ## Changing namespace will cause devices being recreated in NDM app.
+  ## This field is used by traps
```

Comment:
Is this field used by "traps too" or "only traps"? If the latter, would it make more sense under `snmp_traps_config`?

Also probably needs a period at the end of the sentence.

Reply:
Updated the sentence, the field did already exist (but was undocumented). It's used by both the snmp integration and the traps listener. Likely we should recommend user to use this one instead.